.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "katsu"]
```

Comment:
You probably don't need to include katsu in the repo - you could just pull the container (https://github.com/CanDIG/katsu/pkgs/container/candig-katsu)